website/components/features-list/style.module.css

```diff
@@ -0,0 +1,107 @@
+.featureListContainer {
+  background-color: #000;
+  padding-top: 128px;
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
+  color: #fff;
+}
+
+.featureContainer {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  & .featureTextContainer {
+    display: flex;
+  }
+
+  & .imageContainer {
+    max-width: 490px;
+    margin: 0 auto;
+    padding-bottom: 40px;
+
+    & img {
+      max-width: 100%;
+    }
+  }
+
+  @media (--large) {
+    flex-wrap: nowrap;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+
+    & .featureText {
+      width: 488px;
+    }
+
+    & .imageContainer {
+      margin: 0;
+    }
+  }
+}
+
+.featuresContainer {
+  padding-top: 157px;
+  padding-bottom: 394px;
+  display: grid;
+  row-gap: 120px;
+}
+
+.title {
+  composes: g-type-display-1 from global;
+  max-width: 488px;
+  margin: 0;
+}
+
+.listNumber {
```

Comment:
Gets a little tight on mobile.  I'm willing to bet the intention is that these remain square if possible.

<img width="495" alt="CleanShot 2021-09-21 at 11 13 04@2x" src="https://user-images.githubusercontent.com/2105067/134225030-069c12e7-aa1c-43f8-bfdb-fdd2c703b3e1.png">